aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -423,6 +423,9 @@ class TORCH_API OperatorHandle {
     // will be done by the time a typed() handle is acquired.
 #if !defined C10_MOBILE
     operatorDef_->op.assertSignatureIsCorrect<FuncType>();
+    if (fn_has_symint<FuncType>::value) {
+      operatorDef_->op.assertSignatureIsCorrect<typename fn_remove_symint<FuncType>::type>();
+    }
```

Comment:
hmm... it's possible I should have pushed the recursive call into assertSignatureIsCorrect? IDK.

Reply:
That sounds better, so it applies to all the callsites of assertSignatureIsCorrect?